lib/workers/qemu.js

```diff
@@ -57,9 +53,27 @@ module.exports = class QEMUWorker {
       '-smp', '4'
     ])
 
-    childKillEventHandler(this.child, 'exit')
-    childKillEventHandler(this.child, 'uncaughtException')
-    childKillEventHandler(this.child, 'unhandledRejection')
+    this.child.stderr.on('data', (data) => {
+      stderr += data
+    })
+
+    this.child.on('close', () => {
+      if (stderr.length !== 0) {
+        throw new Error(stderr)
```

Comment:
But `stderr` not always means an error, right? A lot of programs use it for debugging info or other informative messages. Can we rely on just the exit code?

Reply:
yes we can.